src/reducers/form.js

```diff
@@ -91,6 +113,19 @@ const formReducer = (state = initialState, action) => {
         }
       }
     }
+    case LOAD_DATA_TO_FORM: {
+      const { payload } = action;
+      const { formName, data } = payload;
+      const form = state[formName]
+      return {
+        ...state,
+        [formName]: {
+          ...form,
+          data:
+            data,
```

Comment:
Em làm như vậy thì lúc lấy ra sẽ phải là

[formName].data.data
-> Sử thành

```suggestion
            ...data,
```

Reply:
Hoặc dùng cú pháp short-hand `data,`